dbms/tests/queries/1_stateful/00077_log_tinylog_stripelog.reference

```diff
@@ -1,10 +1,17 @@
-1
-1
-1
 8873898	12457120258355519194
 8873898	12457120258355519194
 8873898	12457120258355519194
 8873898	12457120258355519194
-1
-1
-1
+AdvEngineID.bin	1	
+CounterID.bin	1	
+RegionID.bin	1	
+SearchPhrase.bin	1	
+UserID.bin	1	
+__marks.mrk	1	
+AdvEngineID.bin	1	
+CounterID.bin	1	
+RegionID.bin	1	
+SearchPhrase.bin	1	
+UserID.bin	1	
+data.bin	1	
+index.mrk	1	
```

Comment:
PS. Why not to add a functional test that will CHECK partition of MergeTree table just in case (and always return success)?